<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>859</width>
    <height>711</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PPOCR</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="centralLayout">
    <item>
     <layout class="QHBoxLayout" name="mainLayout" stretch="20,57">
      <item>
       <widget class="QTreeView" name="folderTreeView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>215</width>
          <height>609</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="contentLayout" stretch="8,3">
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>611</width>
            <height>450</height>
           </size>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="previewTab">
           <attribute name="title">
            <string>图像预览</string>
           </attribute>
           <layout class="QVBoxLayout" name="previewLayout">
            <item>
             <widget class="QLabel" name="imageLabel">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>400</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::Box</enum>
              </property>
              <property name="text">
               <string>图像预览</string>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="resultTab">
           <attribute name="title">
            <string>识别结果</string>
           </attribute>
           <layout class="QGridLayout" name="resultLayout"/>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="logTextEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>611</width>
            <height>150</height>
           </size>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>日志信息...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenFile"/>
   <addaction name="actionOpenFolder"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="actionRec"/>
   <addaction name="actionSave"/>
  </widget>
  <action name="actionOpenFile">
   <property name="text">
    <string>打开文件</string>
   </property>
   <property name="toolTip">
    <string>打开单个文件</string>
   </property>
  </action>
  <action name="actionOpenFolder">
   <property name="text">
    <string>打开文件夹</string>
   </property>
   <property name="toolTip">
    <string>打开文件夹</string>
   </property>
  </action>
  <action name="actionRec">
   <property name="text">
    <string>开始识别</string>
   </property>
   <property name="toolTip">
    <string>开始识别图像</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>保存结果</string>
   </property>
   <property name="toolTip">
    <string>保存识别结果</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
